pkg/handlers/v1/mock_http.go

```diff
@@ -0,0 +1,48 @@
+// Code generated by MockGen. DO NOT EDIT.
```

Comment:
Unless you suffix the file name with `_test.go` such as `mock_http_test.go` then it will be built and included with the normal binary. Please rename this so it is only compiled as part of the tests.

Reply:
Oh, didn't know that. I'll change it, thanks